packages/SwingSet/test/test-xsnap-store.js

```diff
@@ -109,7 +109,7 @@ test('SES bootstrap, save, compress', async t => {
   const zfile = path.resolve(pool.name, `${h}.gz`);
   t.is(
     relativeSize(zfile, snapSize.SESboot),
-    snapSize.compression,
+    0.2,
```

Comment:
cc @dckc. I revised this test to make it work with the new version of SES. I expect the hashes below to change any time we change SES, so that’s fine. I also expect that this relative compression-ratio/change to one-significant-digit detector changing a little is okay too.

Reply:
OK, but I would expect the change in the `snapSize` declaration, not here.

Reply:
For whatever reason, the compression ratio is on the cusp of 0.1 and 0.2, so one of these tests is okay with 0.1 and the other is okay with 0.2. If we need a one-size-fits-all assertion, this’ll have to be refactored to assert a range.

Reply:
oh boring. ok.